### PR TITLE
Fix bug when providing results zip file

### PR DESCRIPTION
**Pull request by simeoncarstens**
I swear I had fixed this already - anyways, this fixes a bug that leads to invalid GCS URIs, at least invalid for `boto3`. Also removes a line of unneeded code.

### CODE REVIEW

File: app/scheduler/chainsail/scheduler/tasks.py [hunk 1]
```diff
@@ -144,7 +144,6 @@ def scale_job_task(job_id, n_replicas) -> bool:
 def get_results_signed_url(job_id):
     logger.info(f"Getting signed URL for results of job #{job_id}...", extra={"job_id": job_id})
     # FIXME: new implementation
-    s3_client, container = get_s3_client_and_container()
```
Comment (steshaw) [inline app/scheduler/chainsail/scheduler/tasks.py:147]: Oh, I see, this is the unnecessary code 👍

**Review by steshaw — APPROVED** [review]
Looks good to me

**Comment by steshaw** [comment]
@simeoncarstens, I'd like to "Rebase and merge" this, but the recent changes to protected branches seem to be preventing me. I don't have access to GitHub Settings. Could you take a look?

**Comment by simeoncarstens** [comment]
@steshaw should work now with recent changes to `CODEOWNERS` (you are one now) :slightly_smiling_face:

**Comment by steshaw** [comment]
There we go. Thanks!

**Pull request merged**